src/trivial.rs

```diff
@@ -0,0 +1,301 @@
+//! Trivial transmutation, content-wise
```

Comment:
This phrasing doesn't sit well with me tbh. Maybe sth like

> Transmutation of trivial objects

> Transmutation of objects consisting of trivial data

or something along those lines?